book/website/collaboration/inclusive-events.md

```diff
@@ -0,0 +1,39 @@
+(cl-inclusive-events)=
+# Hosting Inclusive Events
+
+(cl-inclusive-events-prerequisites)=
+## Prerequisites
+
+| Prerequisite | Importance | Notes |
+| -------------|----------|------|
+| {ref}`cm-lay-summaries` | Helpful | ... |
+| {ref}`cl-organising-conferences` | Helpful | ... |
+| {ref}`cl-event-tools` | Helpful | ... |
+| {ref}`cl-remote-collab-guidelines` | Helpful | ... |
+| {ref}`cl-organising-meetings` | Helpful | ... |
+| {ref}`cl-hybrid-collab-challenges` | Helpful | ... |
+| {ref}`cl-chairing` | Helpful | ... |
+| {ref}`cl-new-community-differences` | Helpful | ... |
+
+(cl-inclusive-events-summary)=
+## Summary
+
+This Chapter discusses planning and hosting events for people with lived experience and acts as a guide for how to design an inclusive event for all attendees. 
+It covers: 
+- planning and logistical considerations
+- guidance on how to evaluate and choose a location
+- considerations and requirement for developing a communications strategy
+- suggestions for how to create an inclusive and accessible agenda
+- accessible technology requirements and reccomendations for hybrid events
+- and provides a checklist that can be tailored to your specific event
+
+
+(cl-inclusive-events-background)=
+## Motivation and Background
+
+Every person should have the opportunity to participate fully and equally in all events and social activities – regardless of their individual ability, gender, age, ethnic or social background. 
+This chapter is meant to serve as a resource to support and guide our community members with the tools to identify the opportunities and challenges in upholding this, and to approach event planning and design with accessibility and inclusion in mind.
+
+Because both internal resources and the events you run can range in size - from from small meetings to massive multi-day conferences - this chapter is not an exhaustive list and instead is meant to provide general guidance about how to design events that are inclusive for people with lived experience. 
```

Comment:
```suggestion
Because both internal resources and the events you run can range in size - from small meetings to massive multi-day conferences - this chapter is not an exhaustive list and instead is meant to provide general guidance about how to design events that are inclusive for people with lived experience. 
```

Reply:
Thanks for catching this Saranjeet!